static/js/components/email/EmailCompositionDialog.js

```diff
@@ -16,7 +17,9 @@ export default class EmailCompositionDialog extends React.Component {
     subheadingRenderer?:        (activeEmail: EmailState) => React$Element<*>,
     closeAndClearEmailComposer: () => void,
     closeEmailComposerAndSend:  () => void,
-    updateEmailFieldEdit:       () => void
+    updateEmailFieldEdit:       () => void,
+    extraUI?:                   (a: string, b: (c: string) => void) => React$Element<*>,
```

Comment:
You're passing in a function here for the `extraUI` prop. The function takes a few arguments, and basically just wraps a react component, passing the props down to the react component. Can you just pass the react component instead, and leave passing props to the component to the `EmailCompositionDialog`? Just wondering if there's a technical reason why that wouldn't work, because if we can I think it's cleaner (the function wrapper is needless indirection).